server/Dockerfile

```
@@ -0,0 +1,11 @@
+FROM python:3.8.12-slim
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE=1
+RUN apt-get update \
+    && apt-get install -y build-essential python3-dev libpq-dev gcc musl-dev \
+    && apt-get clean
+RUN mkdir /code
+WORKDIR /code
+COPY requirements.txt /code/
+RUN pip install -r requirements.txt
+COPY . /code/
```

```suggestion
COPY . /app # copy hết vào /app, còn lại để .dockerignore lo, ngoài ra nó sẽ tự chạy mkdir
WORKDIR /app
CMD ./startup.sh
```
Cần command mặc định, trong đấy `startup.sh` để run gunicorn
Ngoài ra sẽ serve python app thông qua 1 server app là gunicorn chứ ta sẽ ko chơi trần